epix/run_epix.py

```diff
@@ -23,10 +23,10 @@ def main(args, return_df=False, return_wfsim_instructions=False, strax=False):
                                             args['file_name'],
                                             args['debug'],
                                             outer_cylinder=args['outer_cylinder'],
-                                            kwargs_uproot_arrays={'entry_start': args['entry_start'],
-                                                                  'entry_stop': args['entry_stop']},
-                                            cut_by_eventid=args['cut_by_eventid']
-                                            )
+                                            kwargs={'entry_start': args['entry_start'],
+                                                    'entry_stop': args['entry_stop']},
+                                            cut_by_eventid=args['cut_by_eventid'])
+
```

Comment:
can you add a comma at the end and move ) into the next line? Then this is not shown always as something "new/changed" for every updated we do.